configs/python/backend/backend.yaml

```diff
@@ -591,6 +591,13 @@ scanners:
       priority: 5
       options:
         limit: 1000
+  'ScanTlsh':
+    - positive:
+        flavors:
+          - '*'
+      priority: 5
+      options:
```

Comment:
I would add the default score here in the config, even commented out, so it's obvious to those who didn't read that docs that it's an option available.

Reply:
Oversight on my part, good catch.